web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -569,6 +577,30 @@ export const formatters: Formatters = {
         rest['server_hostname'] || rest['addr.local']
       }]: [${error}]`,
   },
+  [eventCodes.SFTP_LINK]: {
+    type: 'sftp',
+    desc: 'SFTP Link',
+    format: ({ user, path, ...rest }) =>
+      `User [${user}] created hard link [${path}] on node [${
+        rest['server_hostname'] || rest['addr.local']
+      }]`,
+  },
+  [eventCodes.SFTP_LINK_FAILURE]: {
+    type: 'sftp',
+    desc: 'SFTP Link Failed',
+    format: ({ user, path, error, ...rest }) =>
+      `User [${user}] failed to create hard link [${path}] on node [${
+        rest['server_hostname'] || rest['addr.local']
+      }]: [${error}]`,
+  },
+  [eventCodes.SFTP_DISALLOWED]: {
+    type: 'sftp',
+    desc: 'SFTP Disallowed',
+    format: ({ user, path, ...rest }) =>
+      `User [${user}] was blocked from creating a SFTP session on node [${
```

Comment:
Wouldn't it be 'a' since SFTP starts with a consonant, 'S'?

Reply:
In case of acronyms, I was taught to use a/an depending on whether the way you pronounce the acronym starts with a consonant or a vowel sound.

https://www.scribbr.com/commonly-confused-words/a-vs-an/#acronyms

> An MRI [em-arr-eye] machine uses magnets to take detailed scans of your organs.
>
> It can be expensive to pay for an SAT [ess-ay-tee] prep course out of pocket.

Reply:
TIL, thanks